Jenkinsfile

```diff
@@ -15,7 +23,10 @@ pipeline {
 			steps {
 				ansiColor('xterm') {
 					nvm(getNodejsVersion()) {
-						sh 'npm run lint'
+						sh '''
+						rm -rf lisk-service/  # TODO
```

Comment:
do we still need this TODO?

Reply:
Yes, otherwise the linting will fail. Not sure what the best solution is. Maybe change the comment to reflect that?